rslib/src/stats/card.rs

```diff
@@ -75,13 +76,14 @@ impl Collection {
             memory_state: card.memory_state.map(Into::into),
             fsrs_retrievability,
             custom_data: card.custom_data,
-            preset: preset.name,
+            preset: preset.name.clone(),
```

Comment:
You can avoid the need for a clone by assigning fsrs_params above this let statement.